debian/autobake-deb.sh

```diff
@@ -30,12 +30,11 @@ then
   # build is not running on Travis or Gitlab-CI
   sed '/-DPLUGIN_COLUMNSTORE=NO/d' -i debian/rules
   # Take the files and part of control from MCS directory
-  if [ ! -f debian/mariadb-plugin-columnstore.install ]
-  then
-    cp -v storage/columnstore/columnstore/debian/mariadb-plugin-columnstore.* debian/
-    echo >> debian/control
-    cat storage/columnstore/columnstore/debian/control >> debian/control
-  fi
+  cp -v storage/columnstore/columnstore/debian/mariadb-plugin-columnstore.* debian/
+  # idempotent, except for the blank line, but that can be tolerated.
+  sed -e '/Package: mariadb-plugin-columnstore/,/^$/d' -i debian/control
+  echo >> debian/control
```

Comment:
This only removes package could it be done thru `git checkout` or store control.old and copy that over debian/control?

Reply:
It remove any `Package: mariadb-plugin-columnstore` that exists in control, however it got there, and assumes the contents of the submodule will add it back.

Reply:
Running `wrap-and-sort` would clean up `debian/control` and remove excess new lines at end of file. However if would fix up other stuff too, which might not be desirable (although mostly is!).

Reply:
@ottok would it also remove duplicate entries? I could see on re-runs this being added multiple times.

Reply:
The build itself is not idempotent. After `dpkg-buildpackage` runs it will leave around a bunch of build artifacts. Re-running `dpkg-buildpackage` itself may fail unless there some explicit cleanup steps, like `git reset --hard; git clean -fdx` between the builds. Those same commands would also clear any `debian/control' etc modifications.

Reply:
Anyway, if the script fails before build starts, this script itself is idempotent now with the change from Daniel.